src/components/Composer.vue

```diff
@@ -1178,6 +1183,8 @@ export default {
 			this.loadingIndicatorTo = loadingIndicator === 'to'
 			this.loadingIndicatorCc = loadingIndicator === 'cc'
 			this.loadingIndicatorBcc = loadingIndicator === 'bcc'
+			this.recipientSearchTerms[loadingIndicator] = term
+
```

Comment:
loadingIndicator should maybe be renamed , for code comprehension since it's used as loading Indicator and an input indicator now . 
Maybe something more general like `addressType` or just `type` would make more sense here